aggregator/src/aggregator/accumulator.rs

```diff
@@ -2,68 +2,46 @@
 
 use super::Error;
 use crate::{
-    datastore::{self, models::BatchUnitAggregation, Transaction},
+    datastore::{self, models::BatchAggregation, Transaction},
     task::Task,
 };
 use derivative::Derivative;
+use futures::future::try_join_all;
 use janus_core::{
     report_id::ReportIdChecksumExt,
     time::{Clock, TimeExt},
 };
-use janus_messages::{Interval, ReportId, ReportIdChecksum, Time};
+use janus_messages::{
+    query_type::{FixedSize, QueryType, TimeInterval},
+    Interval, ReportId, ReportIdChecksum, Time,
+};
 use prio::vdaf::{self, Aggregatable};
 use std::{collections::HashMap, sync::Arc};
 use tracing::debug;
 
-#[derive(Derivative)]
```

Comment:
👍🏻 This was a purposeful change, I decided to restructure since the necessary refactoring was effectively changing most of the file anyway. IMO, it is good to structure a file so that the "main idea" (e.g. primary data structures/functions/code) comes first, and supporting/private code comes after -- this makes it easier to follow for first-time readers.